.github/workflows/ci.yml

```diff
@@ -192,10 +192,20 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
           cache-dependency-path: 'src-ui/package-lock.json'
-      - run: cd src-ui && npm ci
-      - run: cd src-ui && npm run lint
-      - run: cd src-ui && npm run test
-      - run: cd src-ui && npm run e2e:ci
+      - name: Install dependencies
```

Comment:
Now that these include a name, maybe include the blank line for a little visual differentiation?

Reply:
Sure!